models.py

```diff
@@ -438,13 +440,14 @@ def from_form(cls, form: 'PewSheetForm') -> 'Service':
         else:
             secondary_feasts = []
 
-        if form.anthem_title.data or form.anthem_composer.data or form.anthem_lyrics.data:
+        if form.anthem_title.data or form.anthem_composer.data or form.anthem_lyrics.data or form.anthem_translation.data:
```

Comment:
this would be simplified if we could group stuff together

Reply:
Okay, I will do some configurations in my git repo to have the commits for the two different issues on two different branches and then I will try to group the anthem fields together.

Reply:
Okay, I will give it a try!